aws/resource_aws_glue_resource_policy_test.go

```diff
@@ -0,0 +1,155 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/glue"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	awspolicy "github.com/jen20/awspolicyequivalence"
+	"testing"
+)
+
+func CreateTablePolicy(action string) string {
+	return fmt.Sprintf(`{
+  "Version" : "2012-10-17",
+  "Statement" : [
+    {
+      "Effect" : "Allow",
+      "Action" : [
+        "%s"
+      ],
+      "Principal" : {
+         "AWS": "*"
+       },
+      "Resource" : "arn:aws:glue:%s:%s:*"
+    }
+  ]
+}`, action, testAccGetRegion(), testAccGetAccountID())
+}
+
+func TestAccAWSGlueResourcePolicy_Basic(t *testing.T) {
+	resourceName := "aws_glue_resource_policy.test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSGlueResourcePolicyDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSGlueResourcePolicy_Required("glue:CreateTable"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccAWSGlueResourcePolicy(resourceName, "glue:CreateTable"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccAWSGlueResourcePolicy_Required(action string) string {
+	return fmt.Sprintf(`
+data "aws_caller_identity" "current" {}
+
+data "aws_partition" "current" {}
+
+data "aws_region" "current" {}
+
+data "aws_iam_policy_document" "glue-example-policy" {
+  statement {
+    actions = ["%s"]
```

Comment:
I believe that the lint issue is to add more spaces before `=` to align with `resources` below it and it looks good to go.

Reply:
Good catch I really had no clue looking at the logs

Reply:
Fixed it now I get the same exact error as on master 😅 
I'll check again tomorrow if it's fixed in master and rebase if it's fixed

Reply:
oh, good. its ok. ill run test now to verify. it will be fixed on master.